stdlib/Profile/test/runtests.jl

```diff
@@ -344,35 +344,39 @@ end
     @test only(node.down).first == lidict[8]
 end
 
+# FIXME: This test that expects specific information from
+# heap snapshots, which is currently supported in MMTk
```

Comment:
Same thing here.

Seems like you forgot the `@static` before the `if`.

Also, please guard the whole `@testset "HeapSnapshot"`.